packages/kuma-gui/src/app/x/components/x-empty-state/XEmptyState.vue

```diff
@@ -79,18 +76,44 @@
               />
             </slot>
           </template>
-        </KEmptyState>
+        </EntityEmptyState>
       </template>
     </template>
   </XI18n>
 </template>
 
 <script lang="ts" setup>
-import { KEmptyState } from '@kong/kongponents'
+import { KUI_COLOR_TEXT_DECORATIVE_AQUA, KUI_ICON_SIZE_50 } from '@kong/design-tokens'
+import { LocationIcon, AnalyticsIcon, MeshIcon } from '@kong/icons'
+import { EntityEmptyState } from '@kong-ui-public/entities-shared'
+import '@kong-ui-public/entities-shared/dist/style.css'
+
 const props = withDefaults(defineProps<{
   type?: string
 }>(), {
   type: '',
 })
 const slots = defineSlots()
+
+const iconMapping: Record<string, unknown> = {
+  'zone-cps': LocationIcon,
```

Comment:
We recently made a PR (I think it was this one https://github.com/kumahq/kuma-gui/pull/3520) to make our Zone icons all use the same type of icon (i.e. the world looking one). We should either make all zone icons use the LocationIcon as you have done here, or make them all use the World icon. Let me get back to you on this one.

Reply:
<img width="403" alt="Screenshot 2025-02-21 at 1 25 55 AM" src="https://github.com/user-attachments/assets/f44b55ab-e5b8-4d5f-ad2e-f203c43d9582" />
We already are using the World Icon in networks in konnect

Reply:
Ok lets leave this as you have it for now, we can make a decision separately and circle back if necessary 👍